node/testing-page/package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "testing-page",
   "version": "0.1.0",
+  "license": "EPL-2.0",
```

I think our license actually is the following. We also use this in headers in the operator java files and the LICENSE file. WDYT?
```suggestion
  "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
```